tensorflow/lite/micro/kernels/resize_bilinear_test.cc

```diff
@@ -0,0 +1,322 @@
+                             int output_length,
+                             TfLiteResizeBilinearParams* params,
+                             float tolerance = 1e-5) {
+  int inputs_array_data[] = {2, 0, 1};
+  TfLiteIntArray* inputs_array = IntArrayFromInts(inputs_array_data);
+  int outputs_array_data[] = {1, 2};
+  TfLiteIntArray* outputs_array = IntArrayFromInts(outputs_array_data);
+
+  const TfLiteRegistration registration = Register_RESIZE_BILINEAR();
+  micro::KernelRunner runner(registration, tensors, tensors_size, inputs_array,
+                             outputs_array, params);
+
+  TF_LITE_MICRO_EXPECT_EQ(kTfLiteOk, runner.InitAndPrepare());
+  TF_LITE_MICRO_EXPECT_EQ(kTfLiteOk, runner.Invoke());
+
+  for (int i = 0; i < output_length; ++i) {
+    TF_LITE_MICRO_EXPECT_NEAR(expected_output_data[i], output_data[i],
+                              tolerance);
+  }
+
+  return kTfLiteOk;
+}
+
+template <typename T>
+void TestResizeBilinear(const int* input_dims_data, const T* input_data,
+                        const int32_t* expected_size_data,
+                        const T* expected_output_data,
+                        const int* output_dims_data, T* output_data,
+                        TfLiteResizeBilinearParams* params,
+                        float tolerance = 1e-5) {
+  int expected_size_dims_data[] = {1, 2};
+  TfLiteIntArray* input_dims = IntArrayFromInts(input_dims_data);
+  TfLiteIntArray* expected_size_dims =
+      IntArrayFromInts(expected_size_dims_data);
+  TfLiteIntArray* output_dims = IntArrayFromInts(output_dims_data);
+
+  const int output_dims_count = ElementCount(*output_dims);
+
+  // Hack to pass ConstantTensor check in prepare
+  TfLiteTensor t = CreateTensor(expected_size_data, expected_size_dims);
+  t.allocation_type = kTfLiteMmapRo;
+
+  constexpr int tensors_size = 3;
+  TfLiteTensor tensors[tensors_size]{
+      TestCreateTensor(input_data, input_dims),
+      t,
+      TestCreateTensor(output_data, output_dims),
+  };
+
+  TF_LITE_MICRO_EXPECT_EQ(
+      kTfLiteOk,
+      ValidateGoldens(tensors, tensors_size, expected_output_data, output_data,
+                      output_dims_count, params, tolerance));
+}
+
+}  // namespace
+}  // namespace testing
+}  // namespace tflite
+
+TF_LITE_MICRO_TESTS_BEGIN
+
+TF_LITE_MICRO_TEST(HorizontalResize) {
+  const int input_dims[] = {4, 1, 1, 2, 1};
+  const float input_data[] = {3, 6};
+  const int32_t expected_size_data[] = {1, 3};
+  const float expected_output_data[] = {3, 5, 6};
+  const int output_dims[] = {4, 1, 1, 3, 1};
+  float output_data[3];
+
+  TfLiteResizeBilinearParams params = {
+      false, /*align_corners*/
+      false  /*half pixel centers*/
+  };
+
+  tflite::testing::TestResizeBilinear(input_dims, input_data,
+                                      expected_size_data, expected_output_data,
+                                      output_dims, output_data, &params);
+}
+TF_LITE_MICRO_TEST(HorizontalResizeInt8) {
```

Comment:
nit: add newling between the different test cases, here and below.

Reply:
fixed